src/web/app/src/components/SignUp/Forms/Review.tsx

```diff
@@ -106,6 +106,36 @@ const useStyles = makeStyles((theme: Theme) =>
 const Review = connect<{ accountError: string | undefined }, SignUpForm>((props) => {
   const classes = useStyles();
 
+  // Make sure only confirmed channel URL(s) are accepted, after data is "submitted" by "Youtube and Twitch" pages
+  const channelList = props.formik.values.channelUrl;
+  if (channelList.trim() !== '') {
+    // If there is any confirmed channel, match the URL(s) against channel URL list to eliminate unconfirmed ones from the channel URL list
+    const confirmedChannels = props.formik.values.channels;
+    if (confirmedChannels.length > 0) {
+      const confirmedChannelUrls = confirmedChannels.map((channel) => {
+        return channel.feedUrl;
+      });
+
+      let channelListArray = channelList.split(/ +/);
```

Comment:
```suggestion
      let channelListArray = channelList.split(/\s+/);
```

Reply:
Oh, good catch!

Reply:
Same issue here, you' haven't fixed it.  Please don't resolve and ask for another review if you don't fix the previous issues.